CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Changelog
   * pim (@smigopal)
   * pim_group_list (@smigopal)
   * pim_rp_address (@smigopal)
+* Port Channel
+  * interface_port_channel (@saichint)
+  * portchannel_global (@saichint)
```

Comment:
We should be consistent on these names - please choose `port_channel` or `portchannel`.